README.md

```diff
@@ -1,11 +1,13 @@
-# CorrelationVector for Go
```

Comment:
I'd like to see a differentiated title for each repository, so this would be "Correlation Vector for Go"